python/lsst/ts/cRIOpy/GUI/AbstractChart.py

```diff
@@ -0,0 +1,122 @@
+# This file is part of cRIOpy package.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https: //www.lsst.org).
+# See the COPYRIGHT file at the top - level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software : you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.If not, see < https:  // www.gnu.org/licenses/>.
+
+import asyncio
+
+from PySide2.QtCore import Qt
+from PySide2.QtCharts import QtCharts
+from PySide2.QtWidgets import QApplication
+
+from .. import TimeCache
+
+__all__ = ["AbstractChart"]
+
+
+class AbstractChart(QtCharts.QChart):
+    """
+    Parameters
+    ----------
+    updateInterval: `float`, optional
+        Interval for chart redraws responding to append call. Defaults to 0.1
+        second.
+    """
+
+    def __init__(self, parent=None, wFlags=Qt.WindowFlags(), updateInterval=0.1):
+        super().__init__(parent, wFlags)
+
+        self._next_update = 0
+        self.updateInterval = updateInterval
+
+        self.updateTask: asyncio.Future = asyncio.Future()
+        self.updateTask.set_result(None)
+
+    def findAxis(self, titleText, axisType=Qt.Vertical):
+        for a in self.axes(axisType):
+            if a.titleText() == titleText:
+                return a
+        return None
+
+    def findSerie(self, name):
+        """
+        Returns serie with given name.
+
+        Parameters
+        ----------
+        name : `str`
+            Serie name.
+
+        Returns
+        -------
+        serie : `QAbstractSerie`
```

Comment:
It is `QAbstractSeries` instead of `QAbstractSerie`. In addition, based on your code, the doc string is:

```
s : `QAbstractSerie` or None
```

Reply:
Fixed in upcoming commit, thanks.